core/src/main/java/com/bankbazaar/tripmanager/model/Trip.java

```diff
@@ -0,0 +1,45 @@
+package com.bankbazaar.tripmanager.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.sql.Date;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name="trip")
+public class Trip {
+    @Id
+    @Column(name="trip_id")
+    @NotNull(message = "trip_id cannot be null")
+    private Long tripId;
+
+    @Column(name="trip_name")
+    @NotBlank(message = "tripName cannot be blank")
```

Comment:
Dont use @NotBlank here. Column has an attribute nullable set that.

Reply:
This has been resolved